src/widgets/pdfviewwindow.cpp

```diff
@@ -119,7 +119,8 @@ void PdfViewWindow::syncEditorFromBuffer()
     auto buffer = getBuffer();
     if (buffer) {
         const auto url = PathUtils::pathToUrl(buffer->getContentPath());
-        const auto urlStr = url.toString(QUrl::EncodeSpaces);
+        // Solution to ASCII problems, like these file names with these symbols # + &.
+        const auto urlStr = QUrl::toPercentEncoding(url.toString(QUrl::EncodeSpaces));
```

Comment:
Can we just call `toPercentEndonging` of `url`?

Reply:
`toString` cannot be implemented with other parameters, can only use `toPercentEncoding`, adj usage.